code/game/gamemodes/cm_initialize.dm

```diff
@@ -89,9 +89,9 @@ Additional game mode variables.
 
 datum/game_mode/proc/initialize_special_clamps()
 	var/ready_players = num_players() // Get all players that have "Ready" selected
-	xeno_starting_num = CLAMP((ready_players/7), xeno_required_num, INFINITY) //(n, minimum, maximum)
-	surv_starting_num = CLAMP((ready_players/25), 0, 8)
-	merc_starting_num = CLAMP((ready_players/3), 1, INFINITY)
+	xeno_starting_num = min((ready_players/7), xeno_required_num, INFINITY) //(n, minimum, maximum)
+	surv_starting_num = min((ready_players/25), 0, 8)
+	merc_starting_num = min((ready_players/3), 1, INFINITY)
```

Comment:
You do need to remove the infinity. Also this should be max
`CLAMP(x, 1, INFINITY)` is equivalent to `max(x, 1)`